src/main.rs

```diff
@@ -1,3 +1,26 @@
+use crate::car::Car;
+use crate::cars::Cars;
+use crate::game::Game;
+
+mod car;
+mod cars;
+mod game;
+mod view;
+
 fn main() {
-    // implement your logic
+    println!("경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)");
+    let cars: Cars = loop {
+        break match Cars::new(view::read_car_names_input()) {
+            Ok(cars) => cars,
+            Err(e) => {
+                eprintln!("{}", e);
+                continue;
+            }
+        };
+    };
+    let total_rounds = view::read_total_rounds_input();
```

Comment:
I'm not sure how I can improve this part. I wanted to validate the car name input at the Car domain, but the error handling became tricky because I had to re-request for a new input.